Meshtastic/Views/Nodes/EnvironmentMetricsLog.swift

```diff
@@ -28,22 +34,22 @@ struct EnvironmentMetricsLog: View {
 				let locale = NSLocale.current as NSLocale
 				let localeUnit = locale.object(forKey: NSLocale.Key(rawValue: "kCFLocaleTemperatureUnitKey"))
 				let format: UnitTemperature = localeUnit as? String ?? "Celsius" == "Fahrenheit" ? .fahrenheit : .celsius
+				var chartLowTemp: Double = (chartData.min { a, b in a.temperature < b.temperature})?.temperature.localeTemperature() ?? chartDefaultMin
+				var chartHighTemp: Double = (chartData.max { a, b in a.temperature < b.temperature})?.temperature.localeTemperature() ?? chartDefaultMax
+
 				VStack {
 					if chartData.count > 0 {
 						GroupBox(label: Label("\(environmentMetrics.count) Readings Total", systemImage: "chart.xyaxis.line")) {
 							Chart {
 								ForEach(chartData, id: \.time) { dataPoint in
 									AreaMark(
 										x: .value("Time", dataPoint.time!),
-										y: .value("Temperature", dataPoint.temperature.localeTemperature()),
-										stacking: .unstacked
+										yStart: .value("Temperature", dataPoint.temperature.localeTemperature()),
+										yEnd: .value("Minimum Temperature", chartLowTemp)
```

Comment:
This will set the end for the area chart to the minimum temperature so that the area is filled in from the temp to the bottom of the chart. Otherwise the end is 0, meaning the area will be drawn from the y point to 0. For negative temp values, the area will be filled in below 0.